README.md

```diff
@@ -6,5 +6,9 @@ A set of Netuitive analytics configurations, polices, dashboards, and reports th
 
 ### Version next
 
+- Fix operation error in nodes_critical policy, add max statistic for nodes_critical computed metric
+
```

Comment:
Please add a readme entry for the addition of the new policies.

Reply:
@Grumblesmith Done